src/sdk/src/store_sdk.rs

```diff
@@ -493,29 +560,44 @@ impl StoreSDK {
     }
 
     pub async fn get_block_bills(&mut self, height: u64) -> std::result::Result<Vec<Bill>, Status> {
-        let token = self.keep_session().await?;
-        match self.session_pool.get_session_mut(token.as_str()) {
-            Some(session) => {
-                if session.check_session_running() {
-                    let mut client = self.client.as_ref().clone();
-                    let query_bill_key = Some(QueryBillKey {
-                        height,
-                        session_token: token.clone(),
-                    });
-                    let q_req = QueryBillRequest { query_bill_key };
-                    let request = tonic::Request::new(q_req);
-                    let response = client.query_bill(request).await?.into_inner();
-                    session.increase_query(1);
-                    Ok(response.bills)
-                } else {
-                    Err(Status::permission_denied(
-                        "Fail to query bill in this session. Please restart query session",
-                    ))
+        if self
+            .query_session_enabled
+            .load(std::sync::atomic::Ordering::Relaxed)
+        {
+            let token = self.keep_session(false).await?;
+            match self.session_pool.get_session_mut(token.as_str()) {
+                Some(session) => {
+                    if session.check_session_running() {
```

Comment:
check_session_running might return false once `Utc::now().timestamp() - self.session_info.start_time > DEFAULT_SESSION_PERIOD`

should skip check_session_running when query_session_enabled = false

Reply:
Sure, the code block will only be executed when query_session_enabled = true